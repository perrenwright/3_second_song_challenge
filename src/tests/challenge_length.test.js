import React from "react";
import { create } from "react-test-renderer";
import Challenge_length from "../components/challenge_length.js"


describe("Challenge length component", () => {
  test("Matches the snapshot", () => {
    const chalength_button = create(<Challenge_length />);
    expect(chalength_button.toJSON()).toMatchSnapshot();
  });
});

