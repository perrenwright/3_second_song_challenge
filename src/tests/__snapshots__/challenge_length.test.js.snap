// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Challenge length component Matches the snapshot 1`] = `<div />`;
